apps/rosen/app/(bridge)/BridgeTransaction.tsx

```diff
@@ -125,19 +130,36 @@ const BridgeTransaction = () => {
     if (!WalletIcon) return null;
     return (
       <PriceItem sx={(theme) => ({ padding: theme.spacing(1.25, 0.5) })}>
-        <Typography>Wallet</Typography>
-        <Grid container alignItems="center" justifyContent="center" gap={1}>
-          <Avatar
-            sx={{
-              width: 18,
-              height: 18,
-              background: 'transparent',
-            }}
-          >
-            <WalletIcon />
-          </Avatar>
-          <Typography>{selectedWallet.label}</Typography>
-        </Grid>
+        <Typography
+          sx={{
+            padding: '5px',
```

Comment:
We rarely use hard-coded values for spacing. Use `theme.spacing` instead.  
The comment is true for all your other changes, too.